azurerm/resource_arm_application_gateway.go

```diff
@@ -146,6 +146,26 @@ func resourceArmApplicationGateway() *schema.Resource {
 							},
 						},
 
+						"connection_draining": {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"enabled": {
+										Type:     schema.TypeBool,
+										Required: true,
+									},
+
+									"drain_timeout_in_sec": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntBetween(1, 3600),
+									},
```

Comment:
can we add some documentation for these new fields? that lives in this file: 
https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/application_gateway.html.markdown

Reply:
I think this is done